idaes/surrogate/sampling/scaling.py

```diff
@@ -0,0 +1,142 @@
+#################################################################################
+# The Institute for the Design of Advanced Energy Systems Integrated Platform
+# Framework (IDAES IP) was produced under the DOE Institute for the
+# Design of Advanced Energy Systems (IDAES), and is copyright (c) 2018-2021
+# by the software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia University
+# Research Corporation, et al.  All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and
+# license information.
+#################################################################################
+import pandas as pd
+
+class StandardScaler(object):
```

Comment:
Would a better name be `StandardDeviationScalar` or `MeanStdDevScalar`?  Or do you intend this to be the "standard" scalar?

Reply:
I changed this to OffsetScaler